src/core/search/rax_tree.h

```diff
@@ -113,7 +119,8 @@ template <typename V> struct RaxTreeMap {
 
   FindIterator find(std::string_view key) const {
     if (void* ptr = raxFind(tree_, to_key_ptr(key), key.size()); ptr != raxNotFound)
-      return FindIterator{std::pair<std::string_view, V&>(key, *reinterpret_cast<V*>(ptr))};
+      return FindIterator{std::pair<std::string, V&>(std::string(key), *reinterpret_cast<V*>(ptr))};
```

Comment:
I know it's dangerous, but it also means every lookup is a copy

Reply:
Well we already leaked in the tests :smile: I will figure out something and push it. I will merge this and follow up